### PR TITLE
Migrating contrib.cloud to tensorflow_io and contrib.BigTable fixes/enhancement

### PR DESCRIPTION
**Pull request by henrytansetiawan**

### CODE REVIEW

**Comment by yongtang** [comment]
@henrytansetiawan Thanks for the PR! One change you may want to do, is to put the python test files into `tests` directory, with name prefixed of `test_xxx.py`. In this way, the existing tests will be picked up automatically by Travis CI. In `tests` directory, we used emulators for pubsub and some other projects. But any python tests should work I think.

Also, the usage of `py_library` in BUILD files are not necessary. We use bazel to build .so files but for python files, we just use setup.py to automatically pick up any python files and `.so` files automatically. The usage of `py_library` won't hurt though it will not have an impact on the generated whl file or test file.

**Comment by terrytangyuan** [comment]
Any updates on this? @henrytansetiawan

**Comment by henrytansetiawan** [comment]
I am planning to pick up this again, thanks for the pointers and hope we can close all the open issues soon.

**Comment by yongtang** [comment]
Bazel lint issue could be fixed with the following command:
```
docker run -i -t --rm -v $PWD:/v -w /v --net=host golang:1.12 \
    bash -x -e -c 'go get github.com/bazelbuild/buildtools/buildifier && buildifier $(find . -type f \( -name WORKSPACE -or -name BUILD -or -name *.BUILD \))'

```

File: tensorflow_io/bigtable/BUILD [hunk 1]
```diff
@@ -53,6 +71,12 @@ cc_binary(
     srcs = [
         "kernels/test_kernels/bigtable_test_client_test.cc",
     ],
+    copts = [
+        "-pthread",
+        "-std=c++11",
+        "-D_GLIBCXX_USE_CXX11_ABI=0",
```
Comment (yongtang) [inline tensorflow_io/bigtable/BUILD]: @henrytansetiawan This line ` "-D_GLIBCXX_USE_CXX11_ABI=0",` could be removed as it is processed by ./configure.sh automatically now.

**Comment by yongtang** [comment]
@henrytansetiawan All tests pass 👍 🎉 ! I will merge this PR now. It is preferred to add some additional python tests with a Bigtable emulator in the follow up PRs.

**Pull request merged**